lib/splitclient-rb/clients/split_client.rb

```diff
@@ -241,5 +171,112 @@ def valid_client
       end
       SplitIoClient.configuration.valid_mode
     end
+
+    def treatments(key, split_names, attributes = {}, metric = 'sdk.get_treatments')
+      return nil unless SplitIoClient::Validators.valid_get_treatments_parameters(split_names)
+
+      sanitized_split_names = sanitize_split_names(split_names)
+
+      if sanitized_split_names.empty?
+        SplitIoClient.configuration.logger.error('get_treatments: split_names must be a non-empty Array')
```

Comment:
It should warn depending on the method invoked by the customer, so get_treatments could be get_treatments_with_config.
Same applies for the singular treatment call.